### PR TITLE
Alarm in the image display

### PR DESCRIPTION
**Pull request by ArtemAndrushkevich**
error

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (MichailZyusko) [inline .vscode/launch.json:15]: Usually `.vscode` leave in `.gitignore` file

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
     <div>
       <header>
-      <div classNameName="header">
+        
+      <div className="header">
         <div className="conteiner"> 
         <div className="header-line">
                 <div className="header-logo">
-                <img src="logo.png" width="50" height="50" alt="" />
+                <img src="logo.png" width="50" height="50" alt="@" className='logo'/>
```
Comment (MichailZyusko) [inline src/App.js:11]: alt === Logo or App logo. You shouldn't pass here className)

File: src/App.js [hunk 3]
```diff
@@ -24,8 +24,10 @@ function App() {
                 </div>
                 <div className="cart">
                     <a href="@">
-                        <img src="cart.png" width="30" height="30" alt="" />
+                        <img src="cart.png" width="30" height="30" alt="cart" />
+                        
                     </a>
+
```
Comment (MichailZyusko) [inline src/App.js:30]: U can remove all empty lines. Generally it use to separate code blocks

**Review by MichailZyusko — COMMENTED** [review]
Look at this comments!

File: .vscode/launch.json (hunk 1 above)
Comment (ArtemAndrushkevich) [reply .vscode/launch.json:15]: ?

**Pull request merged**